docs/custom-models.md

```diff
@@ -260,9 +118,9 @@ If you want to inspect the extracted data for a different codebase before trying
 
 ### How we secure your data
 
-Your models are private to you, those people that have the sharing links that you generate by choosing **Share completions**, and, those who have access to a [repository with an associated model](#sharing-your-repository-associated-models). 
+Your models are private to you, and, those who have access to a [repository with an associated model](#sharing-your-repository-associated-models). 
 
-All data you send to and receive from the IntelliCode service is transmitted over HTTPS. You must [sign in to Visual Studio](/visualstudio/ide/signing-in-to-visual-studio) in order to communicate with the service. Models can only be retrieved either by the authenticated user who submitted the extracted data for training, by someone they authorized by sharing the link to the model, and by users who can prove they have access to the repository for [repository-associated models](#sharing-your-repository-associated-models). This means that your model and what is learned about your code stays private to you and your intended collaborators.
+All data you send to and receive from the IntelliCode service is transmitted over HTTPS. You must [sign in to Visual Studio](/visualstudio/ide/signing-in-to-visual-studio) in order to communicate with the service. Models can only be retrieved either by the machine which submitted the extracted data for training, or for repo-associated models by users who can prove they have access to the repository for [repository-associated models](#sharing-your-repository-associated-models). This means that your model and what is learned about your code stays private to you and your intended collaborators.
```

Comment:
break this up into multiple lines, several different ideas are presented here.

Reply:
Fixed, thanks Viv